x-pack/plugins/apm/common/search_strategies/correlations/types.ts

```diff
@@ -27,7 +27,7 @@ export interface SearchServiceParams {
   start?: string;
   end?: string;
   percentileThreshold?: number;
```

Comment:
I've seen this call both `percentiles`, `percents` and `percentileThreshold`. Types vary between `number`, `number[]` and `Record<string, number>`. Is it possible to improve the consistency of this?

```suggestion
  percentileThreshold?: number[];
```

Reply:
The variables serve different uses, we cannot make it the same type because of what we have to pass on and get from ES aggregations. I tried to improve the naming:

- Renamed `percents` to `percentileAggregationPercents`, it relates to the `percents` attribute we pass on to the ES percentile aggregation.
- Renamed one of the `percentileThreshold` vars to `percentilesResponseThresholds`. This was misleading because it resembles the original format we receive from the ES (the `Record<string, number>`) we want to isolate and convert to `number`.

154b386